sources/shared-defaults/nvidia-k8s-device-plugin.toml

```diff
@@ -20,11 +20,10 @@ path = "/etc/nvidia-migmanager/nvidia-migmanager.toml"
 template-path = "/usr/share/templates/nvidia-k8s-device-plugin-mig-conf"
 
 [metadata.settings.kubelet-device-plugins.nvidia]
-affected-services = ["nvidia-k8s-device-plugin"]
+affected-services = ["nvidia-k8s-device-plugin", "nvidia-container-toolkit"]
```

Comment:
This needs a migration, is that handled somewhere else?

Reply:
Never mind - this is covered by the blanket metadata removal and replacement on upgrade and downgrade.